runtime/rasdump/dmpagent.c

```diff
@@ -886,7 +886,12 @@ static omr_error_t
 doJavaDump(J9RASdumpAgent *agent, char *label, J9RASdumpContext *context)
 {
 	J9JavaVM *vm = context->javaVM;
-	if ((0 != strcmp(label, "/STDOUT/")) && (0 != strcmp(label, "/STDERR/"))) {
+
+	if (('-' == label[0]) || (0 == strcasecmp(label, J9RAS_STDOUT_NAME))) {
```

Comment:
This matches anything that starts with `'-'`: you probably want to use `strcmp` instead.